test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -68,3 +70,350 @@ def forward(self, x):
                 self.checkGraphModuleNodes(m,
                                            expected_node_occurrence=node_occurrence,
                                            expected_node_list=node_list)
+
+    @skipIfNoX86
+    def test_conv2d_unary_with_quantizer_api(self):
+        class Mod(torch.nn.Module):
+            def __init__(self, inplace_relu: bool = False, use_bias: bool = False) -> None:
+                super().__init__()
+                self.conv = nn.Conv2d(3, 6, (2, 2), stride=(1, 1), padding=(1, 1), bias=use_bias)
+                self.relu = nn.ReLU(inplace=inplace_relu)
+
+            def forward(self, x):
+                return self.relu(self.conv(x))
+
+        inplace_relu_list = [True, False]
+        use_bias_list = [True, False]
+        with override_quantized_engine("x86"):
+            with torch.no_grad():
+                for inplace_relu, use_bias in itertools.product(inplace_relu_list, use_bias_list):
+                    m = Mod(inplace_relu=inplace_relu, use_bias=use_bias).eval()
+                    m_copy = copy.deepcopy(m)
+                    example_inputs = (torch.randn(2, 3, 16, 16),)
+                    # program capture
+                    m, guards = torchdynamo.export(
+                        m,
+                        *copy.deepcopy(example_inputs),
+                        aten_graph=True,
+                    )
+
+                    before_fusion_result = m(*example_inputs)
+                    import torch.ao.quantization._pt2e.quantizer.x86_inductor_quantizer as xiq
+                    quantizer = X86InductorQuantizer()
+                    operator_spec = xiq.get_default_x86_inductor_quantization_config()
+                    quantizer.set_global(operator_spec)
+                    # Insert Observer
+                    m = prepare_pt2e_quantizer(m, quantizer)
+                    after_prepare_result = m(*example_inputs)
+                    m = convert_pt2e(m)
```

Comment:
wondering if you could refactor boilerplate code to a helper function, similar to https://github.com/pytorch/pytorch/blob/main/test/quantization/pt2e/test_quantize_pt2e.py#L215

Reply:
Thanks for the comments. Refactor boilerplate code as in https://github.com/pytorch/pytorch/blob/main/test/quantization/pt2e/test_quantize_pt2e.py#L215.